docs/se/health/01_health.adoc

```diff
@@ -251,4 +251,4 @@ values. Helidon reports both of these to maintain backward compatibility with ol
 versions of MicroProfile Health. This behavior can be disabled by setting
 the property `health.backward-compatible` to `false`, in which case only `"status"`
 is reported. Future versions of Helidon will drop support for older versions of Health,
-so it is recommended to rely on `"status"` instead of `"outcome"` in your applications.
+so it is recommended to rely on `"status"` instead of `"outcome"` in your applications.
```

Comment:
This looks like a white space change. This should be displayed in the diff as "File renamed without changes"